2.4/examples/replica/mongodb-clustered.json

```diff
@@ -119,6 +119,20 @@
               {
                 "name":  "mongodb",
                 "image": "openshift/mongodb-24-centos7",
+                "readinessProbe": {
+                  "timeoutSeconds": 1,
```

Comment:
FTR, timeouts are ignored in Exec probes: https://github.com/kubernetes/kubernetes/issues/26895